config/vufind/searches.ini

```diff
@@ -634,13 +634,21 @@ view=full
 ; geographic points are located. If your dataset is not concentrated in one 
 ; geographic area, it is advised that you pick a default area, and do not use
 ; the entire extent of the map for searching (otherwise the search may be slow).
-; The default coordinates specified below are in decimal degrees, and are 
-; ordered as WENS (west, east, north, south). Ranges of valid values are:
-; -180 to 180 (longitude) and -85 to 85 (latitude). Note, to search from and to
-; the international date line, use west = -179 and east = -180. 
-default_coordinates = "-95, 30, 72, 15"
-; height: Height in pixels of the map selection interface.
-height = 320
+; Options:
+; basemap:   Tileserver URL for basemap. Options are osm-intl (default), stamen-toner,
+;            stamen-terrain, cartocdn-light, or cartocdn-dark. To add an alias, edit the
+;            geo_basemaps.txt file located at /vufind/config/vufind/.
```

Comment:
If we change the other comment, we should remember to make this one match.

Reply:
Will do.